config/crd/bases/redhatcop.redhat.io_groupsyncs.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.0
+    controller-gen.kubebuilder.io/version: v0.3.0
```

Comment:
curious, why did you have to do this?

Reply:
Errors where returned in .status.Conditions in `v0.4.0`. Need to investigate it later

Reply:
FYI: I finally got around to managing status conditions in better way in operator-utils.

Reply:
@raffaelespazzoli any pointer to that? I have a controller which does something similar

Reply:
@davidkarlsen https://github.com/redhat-cop/operator-utils/